app/dependency_file_updaters/node.rb

```diff
@@ -63,7 +63,7 @@ def updated_yarn_lock_content
       SharedHelpers.in_a_temporary_directory do |dir|
         File.write(File.join(dir, "yarn.lock"), @yarn_lock.content)
         File.write(File.join(dir, "package.json"), updated_package_json_content)
-        puts `cd #{dir} && yarn install --ignore-scripts`
+        print `cd #{dir} && yarn install --ignore-scripts >nul 2>&1`
```

Comment:
I think you can get rid of the `print` here entirely - backticks will capture STDOUT by default
```ruby
`cd #{dir} && yarn install --ignore-scripts 2>&1`
```